geometries/vel_earth_los/vel_earth_los_call.py

```diff
@@ -22,14 +22,13 @@ def main():
 
     # Specify time of observation based on interval in kernel(s)
     utc = '2025 JUNE 01 00:00:01'
-    galaxy_targ = '9999000'
 
-    vel_los = vel_earth_los(utc, galaxy_targ)
+    vel_los = vel_earth_los(utc, 'ASP_SLIT_0')
```

Comment:
velocity along line of sight is relevant to the instrument, not the galaxy target.